framework/src/org/checkerframework/framework/type/TypeFromExpressionVisitor.java

```diff
@@ -362,14 +370,16 @@ public AnnotatedTypeMirror visitWildcard(WildcardTree node,
     @Override
     public AnnotatedTypeMirror visitPrimitiveType(PrimitiveTypeTree node,
                                                   AnnotatedTypeFactory f) {
-        // for e.g. "int.class"
+        // Node is the primitive type when primitive class literals are visited.
+        // for e.g. "int" when "int.class" is visited
         return f.fromTypeTree(node);
     }
 
     @Override
     public AnnotatedTypeMirror visitArrayType(ArrayTypeTree node,
                                               AnnotatedTypeFactory f) {
-        // for e.g. "int[].class"
+        // Node is the array type when array class literals are visited.
```

Comment:
Should "Node" be the parameter "node"? Why is this visit method only used for array class literals?